chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -566,6 +602,37 @@ impl PartialWitnessActor {
         validators.sort();
         Ok(validators)
     }
+
+    /// Returns for each code-hash the contract code retrieved from the storage.
```

Comment:
Here does code-hash means just the hash of the code? :)

Reply:
yeah it is not clear, I will reword this. I meant hash of the code.